usaspending_api/awards/v2/filters/filter_helpers.py

```diff
@@ -52,19 +53,35 @@ def date_list_to_queryset(date_list, table):
     for v in date_list:
         # Modified May 2018 so that there will always be a start and end value from combine_date_range_queryset()
 
-        date_type = v.get("date_type", "action_date")
-        if date_type not in ["action_date", "last_modified_date", "date_signed"]:
-            raise InvalidParameterException("Invalid date_type: {}".format(date_type))
-
-        kwargs = {"{}__gte".format(date_type): v["start_date"], "{}__lte".format(date_type): v["end_date"]}
+        date_type_dict = v.get("date_type_dict", {"gte": "action_date", "lte": "action_date"})
+        for date_type in date_type_dict.values():
+            if date_type not in ["action_date", "last_modified_date", "date_signed"]:
```

Comment:
This hard-coded list is going to annoy a future dev somewhere. Is there anywhere else we store this information?

Reply:
Not that I saw. It seemed to be unique to the date filter.